packages/bundle-size/src/utils/readConfig.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const fs = require('fs').promises;
+
+/**
+ * @typedef {{
+ *  webpack(config: import("webpack").Configuration): import("webpack").Configuration
+ * }} Config
+ */
+
+/** @type {Config} */
+const defaultConfig = {
+  webpack: config => config,
+};
+
+const CONFIG_FILE_NAME = 'bundle-size.config.js';
+
+/** @type {Config | undefined} */
+let cache;
+
+/**
+ * @returns {Promise<Config>}
+ */
+async function readConfig() {
+  const configPath = path.resolve(process.cwd(), CONFIG_FILE_NAME);
```

Comment:
It's not important, but I would suggest to use [`find-up`](https://github.com/sindresorhus/find-up) instead as this will allow us to have a global config per a repo and a scoped one for packages.

Reply:
yeah it makes sense I can do it

Reply:
yeah it's works nicely 👍